website/src/pages/404.tsx

```diff
@@ -1,24 +1,27 @@
-import { Box, Button, Center, Link, Text } from "@chakra-ui/react";
+import { Box, Button, Center, Link, Text, useColorMode } from "@chakra-ui/react";
 import { AlertTriangle } from "lucide-react";
 import Head from "next/head";
 import { EmptyState } from "src/components/EmptyState";
 import { getTransparentHeaderLayout } from "src/components/Layout";
 export { getDefaultStaticProps as getStaticProps } from "src/lib/default_static_props";
 
 function Error() {
+  const { colorMode } = useColorMode();
+  const backgroundColor = colorMode === "light" ? "white" : "gray.700";
+  const titleColor = colorMode === "light" ? "blue.500" : "blue.300";
   return (
     <>
       <Head>
         <title>404 - Open Assistant</title>
         <meta name="404" content="Sorry, this page doesn't exist." />
       </Head>
-      <Center flexDirection="column" gap="4" fontSize="lg" className="subpixel-antialiased">
+      <Center flexDirection="column" gap="4" fontSize="lg" className="subpixel-antialiased" bg={backgroundColor}>
```

Comment:
can you try to use the `oa-basic-theme`? does it work?

https://github.com/LAION-AI/Open-Assistant/blob/56b6595d0144a8b4eb1402869e1bdc5744dbc40b/website/src/pages/terms-of-service.tsx#L14

Reply:
Yes it works, thx less code same results.